sig-provider/sig-provider/src/lib.rs

```diff
@@ -3,3 +3,6 @@ mod sources;
 
 pub use aggregator::SourceAggregator;
 pub use sources::{fourbyte, sigeth, SignatureSource};
+
+#[cfg(test)]
+pub use sources::mock;
```

Comment:
Why not to use `mockall` crate?

Reply:
because I didn't know/forgot about it :D

Reply:
Get it🙂 You should probably give it a try

Reply:
done